test/per_file_coverage.sh

```diff
@@ -47,6 +47,9 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/health_checkers:95.9"
 "source/extensions/health_checkers/redis:95.9"
 "source/extensions/quic_listeners:85.1"
+"source/extensions/stat_sinks/common:96.5"
+"source/extensions/stat_sinks/common/statsd:96.5"
+"source/extensions/stat_sinks/graphite_statsd:85.2"
```

Comment:
Are you missing real coverage here? Or just the gcovr excluded issues?

Reply:
It's mostly unreachable branches and closing braces. There's also one method in source/extensions/stat_sinks/common/statsd/statsd.cc that has no coverage in the main branch, I haven't touched it in this PR.

Relevant coverage reports (they're a couple of commits old through):
[source/extensions/stat_sinks/common/statsd/statsd.cc](https://storage.googleapis.com/envoy-pr/b66eb3b/coverage/source/extensions/stat_sinks/common/statsd/statsd.cc.gcov.html)
[source/extensions/stat_sinks/graphite_statsd/config.cc](https://storage.googleapis.com/envoy-pr/b66eb3b/coverage/source/extensions/stat_sinks/graphite_statsd/config.cc.gcov.html)